engine/table/src/main/java/io/deephaven/engine/table/impl/TableDefaults.java

```diff
@@ -234,6 +238,157 @@ default Table formatColumns(String... columnFormats) {
         return updateView(selectColumns);
     }
 
+    private void validateDataBarOptions(String positiveColor, String negativeColor, double opacity, String markerColor)
+            throws IllegalArgumentException {
+        if (positiveColor != null) {
+            for (String color : positiveColor.split(",")) {
+                new Color(color); // Throws if it can't create it
```

Comment:
Instead of doing this same patter in multiple places (`new Color(color); // Throws if it can't create it`), put it in it's own method
```
private void validateColor(String color) {
  // Throws if it can't create it
  new Color(color);
}
```

Then it's clear what you're doing in these cases.

Reply:
https://github.com/deephaven/deephaven-core/pull/4181#discussion_r1293732128

Colin had a different perspective. He didn't think it needed to exist since it is just a longer way to write new Color